### PR TITLE
fix typo and fix comment to FoldConjugateIntoTranspose

### PR DESCRIPTION
**Pull request by Agoniii**

### CODE REVIEW

**Comment by Agoniii** [comment]
@googlebot I signed it!

**Comment by xinan-jiang** [comment]
@ezhulenev 
please help to review the comments for Fold Conjugate in arithmetic_optimizer.cc

File: tensorflow/core/grappler/optimizers/arithmetic_optimizer.cc [hunk 1]
```diff
@@ -2322,7 +2322,7 @@ class FoldTransposeIntoMatMul : public ArithmeticOptimizerStage {
   }
 };
 
-// Fold Transpose into matrix multiplication.
+// Fold Conjugate into Transpose.
```
Comment (sanjoy) [inline tensorflow/core/grappler/optimizers/arithmetic_optimizer.cc]: Probably this comment should just be deleted?  It is only repeating the class name with spaces. :)

**Review by sanjoy — DISMISSED** [review]

**Comment by gbaned** [comment]
@Agoniii Could you please check reviewer comments and keep us posted. Thanks!

**Review by ezhulenev — APPROVED** [review]

**Comment by gbaned** [comment]
@Agoniii Could you please check failed build errors? Thanks!

**Comment by Agoniii** [comment]
@gbaned I only fixed typo in comment.  These build errors are not related to the code change in this PR. Thanks.

**Comment by Agoniii** [comment]
@gbaned One error in Ubuntu Sanity is `ERROR: /tmpfs/src/github/tensorflow/tensorflow/compiler/xla/service/cpu/BUILD:659:12: in deps attribute of cc_library rule //tensorflow/compiler/xla/service/cpu:runtime_single_threaded_fft: target '//tensorflow/compiler/xla:xla_data_cc_proto' does not exist` 
but in [BUILD](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/service/cpu/BUILD#L659) , it is `"//tensorflow/compiler/xla:xla_data_proto"`. The codes are different.
The other error is similar.

**Pull request merged**